TRRandomizerCore/Randomizers/TR2/Shared/TR2EnemyAllocator.cs

```diff
@@ -553,7 +553,7 @@ public void RandomizeEnemies(string levelName, TR2Level level, EnemyRandomizatio
                 skidoo.SetLocation(randomLocation);
             }
         }
-        else
+        else if (Settings.RandomizeItems && Settings.RandomizeVehicles)
```

Comment:
I suppose this means vehicle randomization is a sub-setting of randomize items? It needs both to be on to do it. Is there a way to show that in the UI?

Reply:
True. Actually, this really doesn't belong in this class. The first condition does i.e. adding a red skidoo when we've imported a skidoo man, but moving the existing one should be in the main vehicle rando logic. I will tweak.

Reply:
Updated now. We only add an additional skidoo in the enemy code if applicable, and item rando will move the default skidoo if item and vehicle rando are enabled. So the UI is now accurate as-is.